### ISSUE TITLE
open menu in octovis does not work under Windows

### ISSUE THREAD

**Issue by irvingvasquez** [OctoMap/octomap#191]
Open file dialog windows does not display any file, however running from command line it works, not critital but strange. 

BTW great work with octomap.

**Comment by ahornung** [comment]
Which windows version? Which version of Qt did you use to compile?

**Comment by Chiasung** [comment]
I have the same issue in Ubuntu 16.04. I use Qt 5.9.1 (GCC 5.4.0). 
No matter install from ROS Kinetic package or compile from source are the same.

**Comment by ahornung** [comment]
octovis does not yet work with Qt5 and unfortunately still requires Qt4. I'm surprised that the "open file" dialog is the only problem you face.

**Comment by Chiasung** [comment]
Yes, the open file dialog is the only issue. No problem to open file through the command line.

**Comment by ahornung** [comment]
Closing due to inactivity.
